### DISCUSSION TITLE
backend does not start normally [May 01]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2000] [Discord Threads]
> 🙋‍♂️ **tosmatt**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### looks fine on my side  
![image0](https://cdn.discordapp.com/attachments/969489726102970378/969497886175948810/unknown.png)  
#  
> 👨‍🚀 **wingkwong**  
#### please delete node_modules, run npm install and try start again  
#  
> 🙋‍♂️ **tosmatt**  
####   
![image0](https://cdn.discordapp.com/attachments/969489726102970378/969498776949633075/unknown.png)  
#  
> 🙋‍♂️ **tosmatt**  
#### delete node_modules, run npm install and try start but fail  
#  
> 👨‍🚀 **wingkwong**  
#### wt s your node version?  
#  
> 🙋‍♂️ **tosmatt**  
####   
![image0](https://cdn.discordapp.com/attachments/969489726102970378/969499117225115648/unknown.png)  
#  
> 👨‍🚀 **wingkwong**  
#### can u check if the port can occupied by other processes?  
#  
> 👨‍🚀 **wingkwong**  
#### sth like `lsof -i:8080`  
#  
> 🙋‍♂️ **tosmatt**  
#### I restart system,now it can run~ Thank you  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/969489726102970378)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
